cmake/yodart.cmake

```diff
@@ -13,6 +13,14 @@ install(DIRECTORY ./runtime/ DESTINATION /usr/yoda
         PATTERN "**/*.md" EXCLUDE
         PATTERN "**/*.txt" EXCLUDE
         PATTERN "**/*.cc" EXCLUDE)
+
+install(DIRECTORY ./client/js/ DESTINATION /usr/yoda/client
```

Comment:
What about installing to `/usr/yoda-client`? `/use/yoda` contains runtime sources and we'd keep it separated from client code to maintain access controlling.

Reply:
The origin files:
./staging_dir/target-arm-32in64-linux-gnu/usr/yoda
├── app
│   ├── app-bridge.js
│   ├── default-launcher.js
│   ├── executable-launcher.js
│   └── light-launcher.js
├── app-runtime.js
├── client
│   ├── api
│   │   └── default.json
│   ├── ext-app-entry.js
│   ├── ext-helper.js
│   ├── ext-instrument-entry.js
│   ├── translator-in-process.js
│   └── translator-ipc.js

So, should not place the files to /usr/yoda/client ?